tests/foreman/cli/test_provisioning.py

```diff
@@ -2,15 +2,15 @@
 
 :Requirement: Provisioning
 
-:CaseAutomation: Automated
+:CaseAutomation: Manual
```

Comment:
AFAIK, the Manual, used to refer to stuff that we haven't automated yet but we do test it manually (and manually update polarion). In case i only have the options you mentioned, this will probably fall to `NotAutomated`

Reply:
New guidelines only add longer descriptions to existing values and their meaning. Here is what Polarion has to say (this was probably written by Og or Elyézer):

> * automated - A case that has been automated.
> * manualonly - A case that can not or will not be automated.
> * notautomated - A case that has not yet been automated but likely could be.

Both `ManualOnly` and `NotAutomated` are tested manually - or at least should be.